### PR TITLE
Tiwae Select All Fix, Ripanga Scroll & Border Rendering Fix

### PR DESCRIPTION
**Pull request by nickwmurphy**
1. Refactored the select all control in Tiwae

2. Ripanga header is now sticky again

3. Ripanga borders render on the left and at the bottom (even in the overflow area) now

### CODE REVIEW

File: tables/ripanga/src/Ripanga.jsx [hunk 1]
```diff
@@ -106,7 +106,7 @@ export default class Ripanga extends React.Component {
     window.addEventListener('table/checkAll', this.onCheckAll);
     window.addEventListener('table/checkOne', ({ detail: id }) => this.onRowCheck(id));
     window.addEventListener('table/resize', debouncedResize);
-    window.addEventListener('table/scroll', this.onScroll);
+    window.addEventListener('scroll', this.onScroll);
```
Comment (ben-burlingham) [inline tables/ripanga/src/Ripanga.jsx]: What's the reason for this change? I'm worried other components (anywhere) may dispatch an event with the same name.

File: tables/ripanga/src/Ripanga.jsx [hunk 2]
```diff
@@ -124,7 +124,7 @@ export default class Ripanga extends React.Component {
     window.removeEventListener('table/checkAll', this.onCheckAll);
     window.removeEventListener('table/checkOne', id => this.onRowCheck(id));
     window.removeEventListener('table/resize', debouncedResize);
-    window.removeEventListener('table/scroll', this.onScroll);
+    window.removeEventListener('scroll', this.onScroll);
```
Comment (ben-burlingham) [inline tables/ripanga/src/Ripanga.jsx]: See above

File: tables/ripanga/src/Ripanga.scss [hunk 3]
```diff
@@ -58,19 +58,17 @@ $tableZ: 0;
...
-  &:last-child {
-    border-right-width: 1px;
-  }
 }
 
 .headRow {
   composes: tableRow;
   background: #e7e7e7;
+  border: 1px solid #c3c2c2;
+  border-width: 1px 0 1px 0;
```
Comment (ben-burlingham) [inline tables/ripanga/src/Ripanga.scss]: nb Shorthand `1px 0` is nice

File: tables/tiwae/sandbox/sandbox.jsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { render } from 'react-dom';
+import { render } from 'react-dom'; // eslint-disable-line
```
Comment (ben-burlingham) [inline tables/tiwae/sandbox/sandbox.jsx]: nb I've started using `/* eslint-disable import/no-extraneous-dependencies */` at the top of the file, it's a tad more elegant.

File: tables/ripanga/src/Ripanga.jsx (hunk 1 above)
Comment (nickwmurphy) [inline tables/ripanga/src/Ripanga.jsx]: This was preventing the header from being sticky. I talked to @chrisarevir about this briefly. Maybe there is a better way to fix this?

File: tables/ripanga/src/Ripanga.jsx (hunk 2 above)
Comment (nickwmurphy) [inline tables/ripanga/src/Ripanga.jsx]: Same as above.

File: tables/ripanga/src/Ripanga.jsx (hunk 1 above)
Comment (ben-burlingham) [inline tables/ripanga/src/Ripanga.jsx]: OK just chatted with y'all. :+1:
